File: get_keys.py
import json

def flatten_json(y):
    out = {}

    def flatten(x, name=''):
        if type(x) is dict:
            for a in x:
                flatten(x[a], name + a + '.')
        elif type(x) is list:
            i = 0
            for a in x:
                flatten(a, name + str(i) + '.')
                i += 1
        else:
            out[name[:-1]] = x

    flatten(y)
    return out

with open('client/src/locales/en.json', 'r') as f:
    data = json.load(f)
    flat = flatten_json(data)
    for key in sorted(flat.keys()):
        print(key)

File: get_keys.js
const fs = require('fs');

function flatten(obj, prefix = '') {
  let keys = [];
  for (const key in obj) {
    const newKey = prefix ? \`\${prefix}.\${key}\` : key;
    if (typeof obj[key] === 'object' && obj[key] !== null && !Array.isArray(obj[key])) {
      keys = keys.concat(flatten(obj[key], newKey));
    } else {
      keys.push(newKey);
    }
  }
  return keys;
}

const locales = ['en.json', 'es.json', 'fr.json', 'pt-BR.json'];
locales.forEach(file => {
  const content = JSON.parse(fs.readFileSync(\`client/src/locales/\${file}\`, 'utf8'));
  const keys = flatten(content);
  fs.writeFileSync(\`\${file.split('.')[0]}_keys.txt\`, keys.sort().join('\n'));
});
